cmd/kusk/internal/utils/utils.go

```diff
@@ -0,0 +1,56 @@
+package utils
+
+import (
+	"os"
+	"path"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	kuskv1 "github.com/kubeshop/kusk-gateway/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetK8sClient() (client.Client, error) {
+	scheme := runtime.NewScheme()
+	kuskv1.AddToScheme(scheme)
+	corev1.AddToScheme(scheme)
+
+	config, err := getConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	return client.New(config, client.Options{Scheme: scheme})
+}
+
+func getConfig() (*rest.Config, error) {
```

Comment:
I think there might be some similar logic in `GetKubeConfig()` worth re-using.

Reply:
the one in package k8s returns string 
```
func GetKubeConfig() (string, error) {
```